docs/book/pagination.md

```diff
@@ -412,13 +412,15 @@ our modules:
   <?php if (isset($this->previous)): ?>
     <li class="page-item">
       <a class="page-link" href="<?= $this->url($this->route, [], ['query' => ['page' => $this->previous]]) ?>">
-        &lt;&lt;
+      <span aria-hidden="true">&laquo;</span>
+        <span class="sr-only">Previous</span>
```

Comment:
We should use a minimal example and follow the [documentation from Bootstrap](https://getbootstrap.com/docs/4.4/components/pagination/) here.

```suggestion
      Previous
```